src/account-security.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "SORA Account Security: How to Manage Passphrases Securely | SORA Docs"
+head:
+  - - meta
+    - name: viewport
+      content: "width=device-width, initial-scale=1.0"
+  - - meta
+    - name: description
+      content: "Explore the intricacies of passphrase formats like mnemonics, JSON, and raw seeds. Dive into best practices for safeguarding passwords, utilizing physical and digital storage solutions. Prioritize your digital security today."
+  - - meta
+    - name: keywords
+      content: "passphrases, passwords, mnemonic, JSON, raw seed, best practices, digital security, two-factor authentication, physical storage, digital storage, password managers, social engineering, user interfaces, security protocols"
+  - - meta
+    - name: author
+      content: "SORA network"
+  - - meta
+    - property: og:title
+      content: "SORA Account Security: How to Manage Passphrases Securely"
+  - - meta
+    - property: og:description
+      content: "Explore the intricacies of passphrase formats like mnemonics, JSON, and raw seeds. Dive into best practices for safeguarding passwords, utilizing physical and digital storage solutions. Prioritize your digital security today."
+  - - meta
+    - property: og:url
+      content: "/sora-guide"
+  - - meta
+    - property: og:type
+      content: "article"
+  - - meta
+    - property: og:site_name
+      content: "SORA Tutorial: Account Security: How to Manage Passphrases Securely"
+  - - meta
+    - property: og:locale
+      content: "en_US"
+  - - meta
+    - name: twitter:card
+      content: "summary"
+  - - meta
+    - name: twitter:title
+      content: "SORA Account Security: How to Manage Passphrases Securely"
+  - - meta
+    - name: twitter:description
+      content: "Explore the intricacies of passphrase formats like mnemonics, JSON, and raw seeds. Dive into best practices for safeguarding passwords, utilizing physical and digital storage solutions. Prioritize your digital security today."
+  - - meta
+    - name: twitter:site
+      content: "@sora_xor"
+  - - meta
+    - name: twitter:creator
+      content: "@sora_xor"
+---
+
+# SORA Account Security: How to Manage Passphrases Securely
+
+#### Explore the intricacies of passphrase formats like mnemonics, JSON, and raw seeds. Dive into best practices for safeguarding passwords, utilizing physical and digital storage solutions. Prioritize your digital security today.
```

Comment:
I saw multiple occurrences of British English in this repository. Should we consider changing all of them to conform to it?

Reply:
Yes, we should be consistent

Reply:
Yes, let's keep it British, mate